app/routes/events/components/EventAdministrate/RegistrationTables.js

```diff
@@ -305,8 +312,7 @@ export class RegisteredTable extends Component<Props> {
           </span>
         ),
         sorter: (a, b) => {
-          if (a.feedback > b.feedback) return 1;
-          else return -1;
+          return a.feedback.localeCompare(b.feedback);
         },
```

Comment:
This could be simplified to: 

`sorter: (a, b) => a.feedback.localeCompare(b.feedback),`